relay-system/Cargo.toml

```diff
@@ -14,7 +14,7 @@ futures = { workspace = true }
 once_cell = { workspace = true }
 relay-log = { path = "../relay-log" }
 relay-statsd = { path = "../relay-statsd" }
-tokio = { workspace = true, features = ["signal"] }
+tokio = { workspace = true, features = ["rt", "signal"] }
```

Comment:
Should this be a dev dependency?

Reply:
It's actually required by the implementation in `relay_system::service`, so it's needed to compile the non-dev version of the crate. The reason this didn't show up in CI is because the `rt` feature is implicitly enabled by another dependency in the workspace.